packages/accordions/src/elements/components/Step.spec.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { render } from 'garden-test-utils';
+import { Stepper } from '../Stepper';
+import { Step } from './Step';
+
+describe('Step', () => {
```

Comment:
@austin94, I noticed that in `Tiles`, a single spec file exists that tests `Tiles` and all of its static property components. 

Should I delete all the `.spec` files under the `/components` directory which test static property components, and move these tests over to a single `Stepper.spec.tsx`?